uwp/Pdf-Viewer/Getting-Started.md

```diff
@@ -355,6 +355,67 @@ End Sub
 {% endhighlight %}
 {% endtabs %}
 
+## Saving a PDF document
+
+The `Save` method returns the PDF document along with the changes made (annotations addition and modification) as a stream, which can be saved as a file.
+
+{% tabs %}
+{% highlight c# %}
+
+//Save the PDF document
+Stream pdfDocumentStream = pdfViewerControl.Save();
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The Save method does not save the stamp annotations in the PDF document. Instead, use the `SaveAsync` method to save them.
+
+## Saving a PDF document asynchronously
+
+The PDF Viewer also allows you to save and return the PDF document with the changes as a stream asynchronously using the `SaveAsync` method.
+
+{% tabs %}
+{% highlight c# %}
+
+//Save the PDF document asynchronously 
+Task<Stream> pdfDocumentStream = pdfViewerControl.SaveAsync();
+
+{% endhighlight %}
+{% endtabs %}
+
+## Events to track the loading of PDFs
+
+The PDFViewer control allows you to track the loading of PDFs using the following events. The `DocumentLoaded`` event triggers after the document is properly loaded in the SfPdfViewer. The following code example explains the same.
+
+{% tabs %}
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+
+pdfViewer.DocumentLoaded += PdfViewer_DocumentLoaded;
+
+private void PdfViewer_DocumentLoaded(object sender, DocumentLoadedEventArgs args)  
+{  
+        // Get the total page count of the loaded PDF document. 
+        int pageCount = pdfViewer.PageCount; 
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Dispose the managed resources of PDFViewer
+
```

Comment:
We use the naming as either PDF Viewer or PdfViewer. Please cross check with existing.

<img width="249" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/65169077/fc179b78-f8b3-4b17-9616-65b050e6aa32">

Reply:
Feedback addressed.